clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -0,0 +1,229 @@
+                    single_y_axis(format=SECONDS_FORMAT)),
+                simple_graph(
+                    "API call latency aggregated (1s thresholds)",
+                    'histogram_quantile(0.99, sum(rate(apiserver_request_duration_seconds_bucket{verb!="LIST", verb!="WATCH", verb!="CONNECT"}[5d])) by (le, resource, verb, scope, subresource))',
+                    single_y_axis(format=SECONDS_FORMAT)
+                ),
+            ],
+        ),
+        Row(
+            title="Overall cluster health",
+            height=Pixels(300),
+            panels=[
+                simple_graph("Unhealthy nodes",
+                             'node_collector_unhealthy_nodes_in_zone'),
+                # It's not clear which "Component restarts" shows more accurate results.
+                simple_graph(
+                    "Component restarts", 'sum(rate(process_start_time_seconds[1m]) > bool 0) by (job, endpoint)'),
+                simple_graph(
+                    "Component restarts 2", 'sum(min_over_time(container_start_time_seconds{container_name!="",container_name!="POD"}[2m])) by (container_name)'),
+            ],
+        ),
+        Row(
+            title="etcd",
+            height=Pixels(300),
+            panels=[
+                simple_graph("etcd disk backend commit duration",
+                             'histogram_quantile(0.95, sum(rate(etcd_disk_backend_commit_duration_seconds_bucket[5m])) by (le))', single_y_axis(format=SECONDS_FORMAT)),
+                simple_graph("etcd bytes sent", 'rate(etcd_network_client_grpc_sent_bytes_total[1m])',
+                             single_y_axis(format=BYTES_PER_SEC_FORMAT)),
+                simple_graph(
+                    "etcd lists rate", 'sum(rate(etcd_request_duration_seconds_count{operation="list"}[1m])) by (type)',
+                    single_y_axis(format=OPS_FORMAT)),
+                simple_graph(
+                    "etcd operations rate", 'sum(rate(etcd_request_duration_seconds_count[1m])) by (operation, type)',
+                    single_y_axis(format=OPS_FORMAT)),
+                simple_graph("etcd get latency by type (99th percentile)",
+                             'histogram_quantile(0.99, sum(rate(etcd_request_duration_seconds_bucket{operation="get"}[1m])) by (le, type))',
+                             single_y_axis(format=SECONDS_FORMAT)),
+                simple_graph("etcd get latency by type (50th percentile)",
+                             'histogram_quantile(0.50, sum(rate(etcd_request_duration_seconds_bucket{operation="get"}[1m])) by (le, type))',
+                             single_y_axis(format=SECONDS_FORMAT)),
+                simple_graph("etcd objects",
+                             'sum(etcd_object_counts) by (resource)'),
+            ],
+        ),
+        Row(
+            title="kube-apiserver",
+            height=Pixels(300),
+            panels=[
+                simple_graph("goroutines", 'go_goroutines{job="apiserver"}'),
```

Comment:
Please quote strings consistently. 
Actually this can (and should be) done automatically. For personal project I use [black](https://github.com/python/black) which takes care of formatting (and does it sensibly).